runtime/kusama/src/lib.rs

```diff
@@ -735,7 +736,9 @@ impl pallet_offences::Config for Runtime {
 	type OnOffenceHandler = Staking;
 }
 
-impl pallet_authority_discovery::Config for Runtime {}
+impl pallet_authority_discovery::Config for Runtime {
+	type MaxAuthorities = MaxAuthorities;
```

Comment:
not sure if this actually works this simple. Kusama currently has just under 1,000 authorities, and probably we want to pick a number which allows kusama to scale significantly.